tests/test_main.py

```diff
@@ -384,6 +384,39 @@ def test_attr(self):
         c.x = c.y = tester
         str((c.x.x, c.x.y, c.y.x, c.y.y, c.x.asd, c.y.qwe))
 
+    def test_store_attr_multiline(self):
```

Comment:
This is a good start. Please add some more complicated cases with many attributes spanning many lines. Go a bit crazy.